box-ui/src/components/ts/MarshaLoginComponent.tsx

```diff
@@ -58,6 +58,8 @@ const MarshaLoginComponent: FunctionComponent = () => {
             const options = {
                 url: 'http://127.0.0.1:3001/login',
                 method: method,
+                width: 700,
```

Comment:
Values hardcoded in pixels may be a bit annoying if we wan't each university to choose its own screen. Is it possible to set them in percentages ?

Reply:
Yes